.github/workflows/github_pages.yml

```diff
@@ -1,9 +1,10 @@
 name: Github pages
 
 on:
+  workflow_dispatch: # run on request (no need for PR)
   push:
     branches:
-      - releases/v1.0.0
+      - releases/1.1.0
```

Comment:
is this correct?

Reply:
This is exactly same with the latest commit of the `release/1.1.0` branch. Whether the context is correct is a matter for MLOPS team to determine, but it is correct in terms of merge-back.